editor/edit-post/sidebar/document-outline-panel/index.js

```diff
@@ -24,7 +24,7 @@ const PANEL_NAME = 'table-of-contents';
 function DocumentOutlinePanel( { isOpened, onTogglePanel } ) {
 	return (
 		<DocumentOutlineCheck>
-			<PanelBody title={ __( 'Document Outline' ) } opened={ isOpened } onToggle={ onTogglePanel }>
+			<PanelBody title={ __( 'Table of Contents' ) } opened={ isOpened } onToggle={ onTogglePanel }>
```

Comment:
The label is "Table of Contents", the folder is `document-outline-panel`, the `document-outline-panel/item.js` component is `TableOfContentsItem` 😬

Reply:
The thing is, we already have a `TableOfContents` component but I think they are inverted. The `DocumentOutline` should be `TableOfContents` and vice versa